src/main/java/com/configcat/ConfigCatClient.java

```diff
@@ -44,20 +59,23 @@ private ConfigCatClient(String sdkKey, Builder builder) throws IllegalArgumentEx
                     .build()
                 : builder.httpClient,
                 this.logger,
+                configMemoryCache,
                 sdkKey,
                 !hasCustomBaseUrl
-                    ? builder.dataGovernance == DataGovernance.GLOBAL
-                        ? BASE_URL_GLOBAL
-                        : BASE_URL_EU
-                    : builder.baseUrl,
+                        ? dataGovernance == DataGovernance.GLOBAL
+                            ? BASE_URL_GLOBAL
+                            : BASE_URL_EU
+                        : builder.baseUrl,
                 hasCustomBaseUrl,
                 pollingMode.getPollingIdentifier());
 
         ConfigCache cache = builder.cache == null
                 ? new InMemoryConfigCache()
                 : builder.cache;
 
-        this.refreshPolicy = this.selectPolicy(pollingMode, cache, fetcher, this.logger, sdkKey);
+        this.refreshPolicy = this.overrideBehaviour == OverrideBehaviour.LOCAL_ONLY
```

Comment:
Maybe it's not needed to create the fetcher if the `overrideBehaviour` is `LOCAL_ONLY`

Reply:
done